Lib/warnings.py

```diff
@@ -397,9 +397,13 @@ class WarningMessage(object):
 
     def __init__(self, message, category, filename, lineno, file=None,
                  line=None, source=None):
-        local_values = locals()
-        for attr in self._WARNING_DETAILS:
```

Comment:
Is this the only use of ``_WARNING_DETAILS``? Does it make sense to just get rid of it?